src/table/boot.rs

```diff
@@ -1,10 +1,7 @@
 //! UEFI services available during boot.
 
 use super::Header;
-use crate::proto::{
-    loaded_image::{DevicePath, LoadedImage},
-    Protocol,
-};
+use crate::proto::{device_path::DevicePath, loaded_image::LoadedImage, Protocol};
 use crate::{data_types::Align, proto::media::fs::SimpleFileSystem};
```

Comment:
@GabrielMajeri, the same is happening with this one. That is being used in line 585.